server.js

```diff
@@ -31,11 +28,11 @@ app.get('/version', function (request, response) {
  * @name Calculate
  * @route {POST} /calculate
  * @bodyparam measure - the cqm to calculate against.
- * @bodyparam valueSets - the value sets to use when calculating the measure.
+ * @bodyparam valueSetsByOid - the value sets to use when calculating the measure. Can either be an array, or an object keyed by OID and then version.
```

Comment:
This comment seems to be incorrect now, since if valueSetsByOid is an array an error response is returned.